custom_components/adjustable_bed/support_report.py

```
@@ -0,0 +1,238 @@
+        for handler in root_logger.handlers:
+            if hasattr(handler, "buffer"):
+                # MemoryHandler or similar
+                for record in list(handler.buffer)[-500:]:
+                    if (
+                        DOMAIN in record.name
+                        or "bluetooth" in record.name.lower()
+                        or "bleak" in record.name.lower()
+                    ):
+                        logs.append({
+                            "timestamp": datetime.fromtimestamp(
+                                record.created, tz=timezone.utc
```

Ruff (0.14.13)

Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)